functional-tests/README.md

```diff
@@ -0,0 +1,18 @@
+Functional tests for Transformer's CLI.
+
+They run in a special virtualenv where the `transformer` package, created
+by `poetry build`, is installed.
+
+This directory contains:
+  - a Makefile,
+  - Python scripts with a name starting with `test_`.
```

Comment:
I think that we can just state what test runner is used, the rest of it (test scripts starting with `test_` - here and in the paragraphs below) to me is only a detail of Pytest (its default configuration).